### PR TITLE
Style analysis header based on whether or not it is stuck

### PR DESCRIPTION
**Pull request by danielfdsilva**
Mostly handles the shell. Updating the content is not part of this PR

### CODE REVIEW

**Comment by danielfdsilva** [comment]
Let's get this reviewed before merging the header into main

File: app/scripts/components/analysis/page-hero-analysis.js [hunk 1]
```diff
@@ -173,5 +250,54 @@ PageHeroAnalysis.propTypes = {
...
 };
+
+function useIsStuck(threshold) {
+  const [isStuck, setStuck] = useState(window.scrollY > threshold);
+
+  useEffect(() => {
+    // Check for the observer pixel.
+    // https://mediatemple.net/blog/web-development-tech/using-intersectionobserver-to-check-if-page-scrolled-past-certain-point-2/
+    const pixel = document.createElement('div');
+    pixel.id = 'page-hero-pixel';
```
Comment (nerik) [inline app/scripts/components/analysis/page-hero-analysis.js:264]: Would be good to store `page-hero-pixel` in a constant.

File: app/scripts/components/common/layout-root.js [hunk 2]
```diff
@@ -20,13 +20,15 @@ const Page = styled.div`
   display: flex;
   flex-direction: column;
   min-height: 100vh;
+  overflow-anchor: none;
```
Comment (nerik) [inline app/scripts/components/common/layout-root.js:23]: So cool to learn about this 👍

**Review by nerik — APPROVED** [review]
Looks good!

File: app/scripts/components/analysis/page-hero-analysis.js (hunk 1 above)
Comment (danielfdsilva) [reply app/scripts/components/analysis/page-hero-analysis.js:264]: good call!

**Pull request merged**